lib/Models/Terria.ts

```diff
@@ -1638,6 +1659,8 @@ async function interpretHash(
             message: { key: "parsingStartDataErrorMessage" }
           });
         }
+      } else if (property === "useExperimentalCompareWorkflow") {
+        terria.configParameters.useExperimentalCompareWorkflow = true;
```

Comment:
should this be inside a else if block? Ie could it be possible that we'd want to set useExperimentalCompareWorkflow in combination with any of the other properties in the hash?

Reply:
ah sharing seems to use #start, so when sharing from using #useExperimentalCompareWorkflow, it reverts back to to the original compare

Reply:
The change looks fine to me. The if/else statements are inside a `forEach` loop, so it will correctly process all hash paremeters. The generated share URL currently doesn't capture the flag, which is alright I think.